dali/imgcodec/parsers/pnm.cc

```diff
@@ -17,14 +17,73 @@
 namespace dali {
 namespace imgcodec {
 
+namespace {
+  // comments can appear in the middle of tokens, and the newline at the
```

Comment:
Total nitpick - we usually don't indent the contents of namespaces. I'm fine with it if linter is.

Reply:
Yes, I missed it because the linter check is passing. No idea why though, as I recall it giving me this error once.